devlab/scripts/run_functional_tests.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -e
+set -x
+
+trap 'clean_exit $LINENO $BASH_COMMAND; exit' SIGHUP SIGINT SIGQUIT SIGTERM EXIT
+clean_exit()
+{
+    pushd $WORKSPACE/cloudferry/devlab
```

Comment:
But in this case we can't run this script from devlab manually. It's a simple to define it if not defined.